tools/ci_build/github/azure-pipelines/templates/jobs/win-ci-vs-2022-job.yml

```diff
@@ -35,9 +31,6 @@ parameters:
   type: boolean
   default: true
 
-- name: ORT_EP_NAME
```

Comment:
I am not sure if this parameter is still needed or not. Because now we are not running CUDA C# tests. Maybe we should keep this parameter here and use it correctly.

Reply:
The reference of ORT_EP_NAME got removed in PR #13637 . I think that was an accident.

Reply:
@skottmckay said:
> If you use --build_csharp (we do on line 145), I believe onnxruntime_csharp.cmake will update csharp/Directory.Build.props, which sets which EPs are enabled via 'USE_xxx' defines.
> This is a better setup as it a) allows an arbitrary combination of EPs to be used and b) doesn't use a /p property which is a global value that overrides any values set in csproj files. 

However, it seems something was missing.